guides/common/modules/proc_copying-a-content-view.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Copying_a_Content_View_{context}"]
+= Copying a content view
+
+Use this procedure to copy a content view.
+You can copy a content view in the {ProjectWebUI} or you can use the Hammer CLI to copy an existing content view into a new content view.
+
+[NOTE]
+====
+A copied content view does not have the same history as a regular content view.
+Version 1 of the content view is where a copied content view begins.
```

Comment:
```suggestion
Version 1 of the copied content view begins at the last version of the original content view.
```
Is this it?

Reply:
Yes, thank you because I couldn't figure out the best way to rewrite this. This is how it was written in the solution article.